frontend/views/components/LinkToCopy.vue

```diff
@@ -10,7 +10,6 @@ component.c-wrapper(
     :value='link'
   )
   a.link.has-ellipsis.c-link(
-    :href='link'
```

Comment:
Picking up from [this conversation](https://github.com/okTurtles/group-income-simple/pull/1074#discussion_r656689397):

> As far as I've tested, clicking the invite link still copies it, and deleting the href has only two visible effects:
> 
> - disables the hyperlink context menu in favor of the default menu;
> - and disables link URL display at the bottom of the viewport when the link is hovered.
> 
> Furthermore, I guess it doesn't really make sense for this to be a real hyperlink, since it isn't performing any navigation. It's more like a button that has to be stylized like a hyperlink.
For example, trying to open it in a new tab would display the dashboard without copying anything to the clipboard.
> 
> Therefore I have tried to refactor it as a `button.link.has-ellipsis.c-link`, but the ellipsis no longer worked, so I left the `a` tag for now.

OK, I see what you're saying, however, I really think it should be kept since it is designed to look like a link, not a button.

Also, what is the danger here in leaving it? As far as I can tell, this is not a user-generated link, but an app-generated link, and so it hsould be safe. Therefore it seems to me like what's going on here is dealing with linter errors, than dealing with a real problem. If so, then we should deal with the linter errors somehow without sacrificing useful and expected behavior like being able to right-click on a link and choosing "Copy Link Address".

Just brainstorming here: if the problem is the linter, is there some way we can flag this, perhaps via a directive or something, as being safe? E.g. create a directive like `linter-ignore-safe-link` or `linter-ignore-app-generated-link`?

Reply:
Per further consideration and discussion with @snowteamer on this over on Slack, we've decided to go ahead with removing the `:href` here.

```
Visually, the invite link still looks like an ordinary link even without the href, thanks to the .link CSS class.Agreed this particular link is safe. But determining this fact requires manual verification, and it would be difficult to do it automatically since it's a generated link (the href is not a literal string but a JavaScript expression).So if we keep allowing such generated links, we'll have to manually review them. That's also a valid choice. After all, there are onle a few of them atm, so we could likely manage them just fine.
However it would go against one of the main goals of this issue, namely automation, since you wrote

>   ... We should not have to police our PRs for this, we should have an automated way of dealing with it.

8:48
Also FWIW, Discord invite links and GitHub repo clone links are not generated as real hyperlinks with href, but as plain text in a readonly text input.
So they have no underline and likely no href attribute.
However the whole URL gets selected on mousedown or touchdown, so that users can easily bring up the context menu if they'd rather copy via that menu.
Browsers recognize the text as a URL and display a suitable popup menu accordingly.
```